test/coverband/collectors/delta_test.rb

```diff
@@ -48,4 +48,32 @@ def setup
     results = Coverband::Collectors::Delta.results(mock_coverage(current_coverage))
     assert_equal(current_coverage, results)
   end
+
+  test 'one shot lines results' do
```

Comment:
Assuming we change [transform_oneshot_line_coverage](https://github.com/danmayer/coverband/pull/264/files#r285338414) somewhere else, I think we can test this without mocking following the same approach as the other tests in this file. Let me know what you think.

Reply:
👍